ush/mesoscale/evs_prepare_sref.sh

```diff
@@ -194,7 +203,7 @@ if [ $modnam = prepbufr ] ; then
 
 export output_base=${WORK}/pb2nc
 
- if [ -s ${COMINobsproc}/gfs.${vday}/18/atmos/gfs.t18z.prepbufr ] ; then 
+ if [ -s ${COMINobsproc}/gfs.${vday}/??/atmos/gfs.t??z.prepbufr ] ; then
```

Comment:
@RoshanShrestha-NOAA @PerryShafran-NOAA Can we confirm that changing 18 to ?? here does what we want it to do here?